common/persistence/nosql/nosql_task_store.go

```diff
@@ -115,14 +115,14 @@ func (t *nosqlTaskStore) LeaseTaskList(
 	if selectErr != nil {
 		if storeShard.db.IsNotFoundError(selectErr) { // First time task list is used
 			currTL = &nosqlplugin.TaskListRow{
-				DomainID:        request.DomainID,
-				TaskListName:    request.TaskList,
-				TaskListType:    request.TaskType,
-				RangeID:         initialRangeID,
-				TaskListKind:    request.TaskListKind,
-				AckLevel:        initialAckLevel,
-				LastUpdatedTime: now,
```

Comment:
nit: consider change `now` to be a more descriptive name like `currentTimeStamp`